src/app/migrations/password/password.4.core.txt

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+@Component({
```

Comment:
Angular demo both fields are linked together by binding `input` but here there are not - so this is creating a bit strange demo to show ?! 

What do we want to show here?

Reply:
1st one - shows text at the bottom
2nd one - shows an error state.

Reply:
Both side of the example don't do the same thing and show different things so its hard for someone to understand why and how things work:

* Angular demo: Both inputs are linked together with a single model object - typing to one update the other one - This is not the same on the core side.
* Core demo error message is always visible no matter what (no validation) but Angular demo handle and show/hide the error message
* Core mention layout but that is not the same one for Angular example. 
* When using `[(ngModel)]` there is almost no need of using `name` in attributes (simplify the examples)
* Angular provides validation rules but Core example does not